apps/wakustealthcommitments/README.md

```diff
@@ -0,0 +1,38 @@
+# wakustealthcommitments
+
+This application/tool/protocol is used to securely communicate requests and responses for the [Stealth Address Scheme](https://eips.ethereum.org/EIPS/eip-5564)
+
+Uses TWN config as default, and content topic: `/wakustealthcommitments/1/app/proto`
+
+## Usage
+
+1. Clone the erc-5564-bn254 repo and build the static lib
+```sh
+gh repo clone rymnc/erc-5564-bn254
+cd erc-5564-bn254
+cargo build --release --all-features
+cp ./target/release/liberc_5564_bn254.a <path-to-nwaku>
+```
+
+> ![NOTE]
+> This static library also includes the rln ffi library, so you don't need to build it separately.
+> This is because using both of them separately brings in a lot of duplicate symbols.
+
+2. Build the wakustealthcommitments app
+```sh
+cd <path-to-nwaku>
+source env.sh
+nim c --out:build/wakustealthcommitments  --verbosity:0 --hints:off -d:chronicles_log_level=INFO -d:git_version="v0.24.0-rc.0-62-g7da25c" -d:release --passL:-lm --passL:liberc_5564_bn254.a --debugger:native apps/wakustealthcommitments/wakustealthcommitments.nim
```

Comment:
Could have been `make wakustealthcommitments` here, but I think this should remain like this as I suggest removing the Make target in any case (as it will be temporary until we incorporate stealth commitment protocol into the node itself).

Reply:
It couldn't have, without setting up a proper pipeline for the erc-5564-bn254 static lib.
removing the makefile entry in any case